src/style.css

```diff
@@ -0,0 +1,186 @@
+body {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 50px;
+}
+
+.header-logo {
+  padding-top: 5px;
+}
+
+.site-navigation-list {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.site-navigation-item {
+  text-transform: uppercase;
+  font-family: "Roboto", sans-serif;
+  color: #000;
+  font-size: 12px;
+  text-decoration: none;
+  font-weight: 500;
+  line-height: 60px;
+
+  display: block;
+  position: relative;
+  margin-left: 20px;
+}
+
+.site-navigation-item:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  margin-left: 0;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00acdc;
+}
+
+.container {
+  width: 944px;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.item-card {
+  width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  margin-right: 48px;
+  margin-bottom: 50px;
+
+  font-family: "Roboto", sans-serif;
+}
+
+.item-card:nth-child(4),
+.item-card:nth-child(8) {
+  margin-right: 0;
+}
+
+.item-card {
+  width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
```

Comment:
add font style to body tag, it's inherited

Reply:
font style to body was add, but i don't remove style to class,  cos i have 5 mistake in test, when remove

Reply:
you can write `font-family: inherit` in this case